src/domain/redeem/exec/execute.velora.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Executes a Leverage Token redeem using `redeemWithVelora`, powered by Wagmi actions.
+ *
+ * This function uses Velora's special redeem function that takes Velora-specific parameters
+ * including the veloraAdapter address, augustus address, offsets, and swapData.
+ *
+ * Invariants/behavior:
+ * - Sends a single transaction to redeem leverage tokens for collateral using Velora
+ * - Does NOT perform ERC-20 approvals or wait for approvals — the UI is responsible for the Approve step.
+ * - Contract addresses and active chain are inferred from the Wagmi config; no chainId parameter.
```

Comment:
need to pass in chain id to handle if user is on a different chain but the ui flow is for a lev token on a diff chain:

eg:

user is on base and clicks on mainnet lev token and goes thru that redeem flow